packages/ios-client/Sources/FishjamClient/utils/AnyJson.swift

```diff
@@ -230,3 +236,13 @@ public struct AnyJson: Codable {
         var stringValue: String { return key }
     }
 }
+
+extension String {
```

Comment:
Are those really better than just using `AnyJson(from: )`? Seems a little bit like an overkill

Reply:
Fair point, I just thought it looks better but you're right.